scripts/combine/setupCombine.py

```diff
@@ -261,10 +279,18 @@ def setup(args,xnorm=False):
     if args.theoryAgnostic:
         logger.error("Temporarily not using mass weights for Wtaunu. Please update when possible")
         signal_samples_forMass = ["signal_samples"]
+    if constrainMass and args.doStatOnly:
```

Comment:
There are two "if" statements with same condition in this line and few lines afterwards